spk/domoticz/src/dsm-control.sh

```diff
@@ -16,7 +17,24 @@ PORT="8084"
 
 start_daemon ()
 {
-    su - ${USER} -c "${DOMOTICZ} -www ${PORT} -approot ${INSTALL_DIR}/ -dbase ${DB_FILE} &> $LOGFILE & echo \$! > ${PID_FILE}"
+    insmod /lib/modules/usbserial.ko > /dev/null
+    insmod /lib/modules/ftdi_sio.ko >/dev/null
+    insmod ${INSTALL_DIR}/modules/cp210x.ko >/dev/null
+    insmod ${INSTALL_DIR}/modules/pl2303.ko >/dev/null
+
+    if [ `/bin/get_key_value /etc.defaults/VERSION buildnumber` -ge "5004" ]; then
```

Comment:
It will take a couple of weeks before I can do any changes to code. How about publishing this first, then remove it in a new PR for later version of Domoticz?